packages/react-native-bots/dangerfile.js

```diff
@@ -109,4 +109,6 @@ if (isMergeRefStable) {
   });
 }
 
+// Ensures that eslint is run from root folder and that it can find .eslintrc
+process.chdir('../../');
 eslint.default();
```

Comment:
Would it also work if you instead set `options.cwd`?

[Documentation: https://eslint.org/docs/latest/developer-guide/nodejs-api](https://eslint.org/docs/latest/developer-guide/nodejs-api#-new-eslintoptions)
```suggestion
eslint.default({
  // Ensures that eslint is run from root folder and that it can find .eslintrc
  cwd: '../../',
});
```

Reply:
Yeah, that was my first approach too @yungsters but unfortunately `@sgtcoolguy/danger-plugin-eslint` only allows us to change the `baseConfig`.

https://github.com/sgtcoolguy/danger-plugin-eslint/blob/master/src/index.ts#L24

![image](https://user-images.githubusercontent.com/11707729/195842975-6adcad76-545c-42c5-9dfa-3c3ed7226655.png)

Reply:
Oh no… thanks for sharing. And yes, as @cipolleschi shared, thanks for fixing! I suspect this was one of the issues that blocked me from making progress on https://github.com/facebook/react-native/pull/34401. 🥰